Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetBillingCollectionUITests.swift

```diff
@@ -20,6 +20,24 @@ final class PaymentSheetBillingCollectionUITests: XCTestCase {
         app.launch()
     }
 
+    var cardInfoField: XCUIElement { app.staticTexts["Card information"] }
+    var contactInfoField: XCUIElement { app.staticTexts["Contact information"] }
+    var fullNameField: XCUIElement { app.textFields["Full name"] }
+    var nameOnCardField: XCUIElement { app.textFields["Name on card"] }
+    var emailField: XCUIElement { app.textFields["Email"] }
+    var phoneField: XCUIElement { app.textFields["Phone"] }
+    var billingAddressField: XCUIElement { app.staticTexts["Billing address"] }
+    var countryField: XCUIElement { app.textFields["Country or region"] }
+    var line1Field: XCUIElement { app.textFields["Address line 1"] }
+    var line2Field: XCUIElement { app.textFields["Address line 2"] }
+    var cityField: XCUIElement { app.textFields["City"] }
+    var stateField: XCUIElement { app.textFields["State"] }
+    var zipField: XCUIElement { app.textFields["ZIP"] }
+    var checkoutButton: XCUIElement { app.buttons["Checkout (Complete)"] }
+    var payButton: XCUIElement { app.buttons["Pay $50.99"] }
+    var successText: XCUIElement { app.alerts.staticTexts["Success!"] }
+    var okButton: XCUIElement { app.alerts.scrollViews.otherElements.buttons["OK"] }
```

Comment:
Shouldn't these all be private?

Reply:
These are tests, so access level doesn't really matter, but yeah, they can be private. Changed.